src/templates/static/why_us.jsx

```diff
@@ -87,10 +87,10 @@ const WhyUs = () => (
                         </div>
                         <div className='gr-4 gr-12-m'>
                             <ul className='checked'>
-                                <li>{it.L('Find the right trade for every market condition.')}</li>
+                                <li>{it.L('Make a trade prediction for every possible market condition.')}</li>
                                 <li>{it.L('Execute your trade in seconds.')}</li>
                                 <li>{it.L('Monitor your portfolio in real time.')}</li>
-                                <li>{it.L('Sell back trades at market prices.')}</li>
+                                <li>{it.L('Sell back trades at market prices. (Restrictions may apply)')}</li>
```

Comment:
typography is wrong - you can't start a parenthesis after a full stop, and no need for a capital after the paranthesis